test-toolkit/conftest.py

```diff
@@ -94,14 +139,16 @@ def accelerator_type(request):
     return request.config.getoption('--accelerator-type')
 
 
-@pytest.fixture(scope='session')
-def docker_image(docker_base_name, tag):
-    return '{}:{}'.format(docker_base_name, tag)
+@pytest.fixture(name='docker_registry', scope='session')
+def fixture_docker_registry(aws_id, region):
+    return '{}.dkr.ecr.{}.amazonaws.com'.format(aws_id, region) if aws_id else None
```

Comment:
fine for now because I don't think we're running these tests in all regions, but "amazonaws.com" won't work everywhere

Reply:
As of now I don't think we have any region specific functionality in our toolkit logic, so i don't think we need to run it in more than one region right now.